### DISCUSSION TITLE
Groovy Gluon 7.0.13064

### DISCUSSION THREAD

**Discussion by gingervik** [BeamMW/beam#1889] [Announcements]
_TL;DR Release Highlights:_

The hard fork will take place at block 1 670 000 on Testnet. To continue using your funds please upgrade your wallets prior to reaching the height!
* Numerous infrastructure improvements so that BEAM wallet (and Dapps in particular) work even more seamlessly and intuitively than before (20+ tasks in total - see in details [here](https://github.com/BeamMW/beam/projects/37))

_Improvements:_
Plentiful user-oriented improvements have been implemented, including:

* Clearer notifications/errors/confirmations for users, so you won't miss anything. 
* Implemented IPFS integration. [Support Documentation](https://github.com/BeamMW/beam/wiki/BEAM-IPFS-Support).
* Added support of HFTx (High-frequency transactions). 
* Ability to recover a web wallet from a mobile recovery file.
* Wallet API users can now sign user messages correctly.

_Known limitations:_

* When using active laser beam channels the CLI wallet should not be offline for over 24 hours
* Once generated, do not offer the same offline address to multiple senders of payments

_Points to mention:_

* Upon using Atomic Swaps with ETH, DAI, WBTC, and USDT, users will have to create an account on [Infura](https://infura.io/).
* The sender of funds to a receiver's Public Offline Address might be able to know that those funds are later being used by the receiver. Beam wallet and Beam explorer do not expose such functionality but it's possible to derive this information by analyzing the Blockchain.

**Verifying release binaries**
*PGP key:* "A6C6 2C9F 5593 1860 AFF8 3BBB 997E AB9F 3DF7 F375"

1. Go to https://keybase.io/beammw and get the Public PGP key.
2. Download the signed binary file (with extension .asc) you want to verify.
3. Verify the signature using PGP Suite or any other PGP verification tools

<hr /><em>This discussion was created from the release <a href='https://github.com/BeamMW/beam/releases/tag/beam-7.0.13064'>Groovy Gluon 7.0.13064</a>.</em>
